example/ios/Podfile.lock

```diff
@@ -10,27 +10,27 @@ PODS:
     - DatadogInternal (= 2.18.0)
   - DatadogRUM (2.18.0):
     - DatadogInternal (= 2.18.0)
-  - DatadogSDKReactNative (2.4.3):
+  - DatadogSDKReactNative (2.4.4):
     - DatadogCore (~> 2.18.0)
     - DatadogCrashReporting (~> 2.18.0)
     - DatadogLogs (~> 2.18.0)
     - DatadogRUM (~> 2.18.0)
     - DatadogTrace (~> 2.18.0)
     - DatadogWebViewTracking (~> 2.18.0)
     - React-Core
-  - DatadogSDKReactNative/Tests (2.4.3):
+  - DatadogSDKReactNative/Tests (2.4.4):
     - DatadogCore (~> 2.18.0)
     - DatadogCrashReporting (~> 2.18.0)
     - DatadogLogs (~> 2.18.0)
     - DatadogRUM (~> 2.18.0)
     - DatadogTrace (~> 2.18.0)
     - DatadogWebViewTracking (~> 2.18.0)
     - React-Core
-  - DatadogSDKReactNativeSessionReplay (2.4.3-alpha.0):
+  - DatadogSDKReactNativeSessionReplay (2.4.4):
     - DatadogSDKReactNative
     - DatadogSessionReplay (~> 2.18.0)
     - React-Core
-  - DatadogSDKReactNativeSessionReplay/Tests (2.4.3-alpha.0):
```

Comment:
should it be `2.4.4-alpha.0`? Or we are promoting it out of alpha?

Reply:
No. Not a blocker, but let me update the Podfile.lock for clarity.

Reply:
Fixed it!